src/open_simplex.rs

```diff
@@ -17,16 +17,20 @@
 //! http://uniblock.tumblr.com/post/97868843242/noise
 
 use num_traits::Float;
+use std::ops::{Add};
 
 use {gradient, math, Seed};
 
 const STRETCH_CONSTANT_2D: f64 = -0.211324865405187; //(1/sqrt(2+1)-1)/2;
 const SQUISH_CONSTANT_2D: f64 = 0.366025403784439; //(sqrt(2+1)-1)/2;
 const STRETCH_CONSTANT_3D: f64 = -1.0 / 6.0; //(1/Math.sqrt(3+1)-1)/3;
 const SQUISH_CONSTANT_3D: f64 = 1.0 / 3.0; //(Math.sqrt(3+1)-1)/3;
+const STRETCH_CONSTANT_4D: f64 = -0.138196601125011; //(Math.sqrt(4+1)-1)/4;
```

Comment:
Is it possible to use the equation here to calculate it instead of hard-coding the value?